README.md

```diff
@@ -5,6 +5,7 @@
 [![Latest release (latest by date)](https://img.shields.io/github/v/release/splunk/ta_cisco_webex_add_on_for_splunk?label=Latest%20Release)](https://github.com/splunk/genesys_cloud_ta/releases)
 [![License](https://img.shields.io/badge/License-Apache_2.0-yellow.svg)](https://opensource.org/licenses/Apache-2.0)
 
+Find more details in our [documentation](https://splunk.github.io/ta_cisco_webex_add_on_for_splunk/)
 > **Cisco Webex Add-on for Splunk** is an Add-on to pull in data from _[Webex REST API](https://developer.webex.com/docs/basics)_ to Splunk.
 
 Here are the endpoints and their mapping soucetypes.
```

Comment:
Could you please update this table to the latest version? You can find it at https://github.com/splunk/ta_cisco_webex_add_on_for_splunk/blob/main/package/README.md#here-are-the-endpoints-their-corresponding-source-types-and-the-required-scopes